templates/default/voipaccount/voipaccountbillinglist.html

```diff
@@ -38,6 +38,70 @@ <h1>{$layout.pagetitle}</h1>
         {assign var="number_of_table_columns" value="11"}
     </colgroup>
     <thead>
+        <tr VALIGN="MIDDLE">
```

Comment:
Takie rzeczy to tragedia. Dalej już nawet nie przeglądam :)

Reply:
@chilek ja przeniosłem tylko obecny kod nie dotykając jego treści....

Reply:
Czy przy zmianie kolejności elementów powinienem modyfikować ich kod?

Reply:
@chilek ten kod działa dobrze, mimo że nie spełnia obecnych norm - nie widzę sensu żeby go przepisywać na nowe standardy. A jednak zmiana miejsca filtrów z nagłówkami tabeli będzie pomocna.

Reply:
???

Reply:
@chilek dopóki mi tego nie wyjaśnisz nie będę w stanie wiedział jakiej strategii oczekujesz.

Reply:
@interduo tylko dlaczego mam Ci to wyjaśniać, skoro google szybciutko znajduje specyfikację HTML? :) Proszę bardzo:
https://html.spec.whatwg.org/multipage/tables.html#the-tr-element

Reply:
@chilek to nie mój kod - ja tylko bloki kodu zamieniłem miejscami.

Nie powinienem tak robić - bez refaktoringu tego co zamieniam?

Reply:
> @chilek to nie mój kod - ja tylko bloki kodu zamieniłem miejscami.

Czy to  istotne czyj? Jak już coś zmieniamy to przynajmniej to co zmieniamy róbmy lepiej niż było.

Reply:
Moje zrobienie lepiej polega właśnie na zamianie typu elementów.
Czy może to być zrobione jako wyizolowana czynność czy musi się łączyć z refaktoringiem tych elementów?

Reply:
Już Ci piszę dlaczego o tym pomyślałem - wiem, że tworzysz mini framework jeśli chodzi o frontend. Uważam, że nie ma sensu przebudowywać fragmentu formularza po to by potem go przebudować ponownie na inny nieaktualny wzór. 

Stąd jeśli powiesz ze trzymamy się zasady, że gdy zamieniamy elementy miejscami należy zrobić refaktoring obu elementów - to oleje tą zmianę elementów miejscami w tym PR bo nie jest krytyczna i może zaczekać na nowy wzór {boxów}. Rozumiesz to?